### DISCUSSION TITLE
clarification needed on reduce function

### DISCUSSION THREAD

**Discussion by SunilAnandatheertha** [mtex-toolbox/mtex#2221] [Ask Anything] [answered]
Hi, we can change pixel resolution using the reduce command. Could someone provide some information on the interpolating scheme used in MTEX in mapping EBSD pixel properties such as Euler angles etc, from the old grid onto the new grid. 

Thank you.

**Comment by ralfHielscher** [comment, marked as answer]
There is no interpolation at all. `ebsd.reduce` is a very simple function which simply throws away some of the data. It is mainly intended for a brief overview of very large EBSD data.
Ralf.

**Reply by SunilAnandatheertha** [reply]
Ah great, thank you :)
